spec/features/node_pages/notes_table_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+describe 'node pages' do
+  describe '#show notes table' do
+    subject { page }
+
+    before do
+      login_to_project_as_user
+
+      node = create(:node)
+      @note = create(:note, node: node, text: "#[Title]#\nNote1\n\n#[Description]#\nn/a\n#[Extra]#\nExtra field")
+      visit node_path(node, tab: 'notes-tab')
+    end
+
+    it 'displays column controls for Title, Created, Created by, Updated' do
```

Comment:
very similar to the `spec/features/issue_pages/table_spec.rb` specs. Maybe dry up with shared examples?

Reply:
👍